src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor

```diff
@@ -1,14 +1,17 @@
 ﻿@using DMPowerTools.Core.Models;
 
-<MudItem xs="1">
-    <MudText Typo="Typo.h6" Color="ModifierColor">@Modifier</MudText>
+<MudItem xs="3" sm="2" md="1">
+    <MudText Typo="Typo.h5" Color="ModifierColor">@Modifier</MudText>
     <MudText Typo="Typo.h5">@Name</MudText>
-    <MudText Typo="Typo.subtitle1">@RawPoints</MudText>
+    <span>
+        <MudNumericField @bind-Value=@RawPoints ReadOnly=@ReadOnly Class="mt-0" />
+    </span>
 </MudItem>
 
 @code {
     [Parameter, EditorRequired] public int RawPoints { get; set; }
     [Parameter, EditorRequired] public string Name { get; set; }
+    [Parameter] public bool ReadOnly { get; set; } = true;
```

Comment:
Did you want to hook this up to the Import as `False` and make sure it updates okay? Or were you thinking another card?

Reply:
I was thinking that would be done with #20. Might need one more story too to allow editing after a creature has been added.